src/connection.ts

```diff
@@ -2,9 +2,7 @@ import { ArangoError, HttpError } from "./error.ts";
 import { ArangojsResponse, createRequest, isBrowser } from "./util/request.ts";
 import { sanitizeUrl } from "./util/sanitizeUrl.ts";
 import { Errback } from "./util/types.ts";
-import X3LinkedList from "https://dev.jspm.io/x3-linkedlist";
-// @ts-ignore
-const { LinkedList } = X3LinkedList;
+import { LinkedList } from "https://github.com/Tnifey/x3-linkedlist/raw/master/src/index.ts";
```

Comment:
jspm version of x3-linkedlist throws error 'cause there are no reference to type declarations